certbot-apache/tests/testdata/gentoo_apache/apache/apache2/vhosts.d/00_default_ssl_vhost.conf

```diff
@@ -58,7 +58,7 @@ Listen 443
 	# PEM encoded CA certificates which form the certificate chain for the
 	# server certificate. Alternatively the referenced file can be the same as
 	# SSLCertificateFile when the CA certificates are directly appended to the
-	# server certificate for convinience.
+	# server certificate for convenience.
```

Comment:
This typo should not be fixed, because this file is copied verbatim from Apache.